cfy_manager/components/restservice/config/cloudify-rest.conf

```diff
@@ -12,6 +12,7 @@
     ldap_domain: '{{ restservice.ldap.domain }}',
     ldap_is_active_directory: {{ restservice.ldap.is_active_directory }},
     ldap_dn_extra: '{{ restservice.ldap.dn_extra }}',
+    ldap_timeout: 30.0,
```

Comment:
We need a `config.yaml` entry for this.

Reply:
i think he added one

Reply:
@yarivgiga nope

Reply:
yeah i see now, i confused it with config.py